Agregar notifcaciones

sebasjm opened the pull request:
Alerta una hora antes sobre la próxima meetup.
Se podría trabajar sobre esta idea para mejorarlo con opciones para:
* configurar con cuanto tiempo de antelación notificar
* suscribirse a notificaciones de ciertos meetups y no de otros

Testeado con Android (ver attach) y web (require que por lo menos la página quede abierta)

Este PR está basado en #10

<img src="https://cloud.githubusercontent.com/assets/161916/26339109/cc5daeee-3f5b-11e7-8bf8-9e273e76ce2f.png" width="256" alt="screenshot_2017-05-23-01-52-47"></img>

sebasjm committed:
add notification

sebasjm committed:
android notification

sebasjm committed:
avisar una hora antes

gianu commented on src/js/modules/notification.js:
```diff
@@ -0,0 +1,66 @@
+    bellElement.classList.add('active')
+    const event = months[0].events[0]
+    const diff_millis = moment(event.date).diff(moment()).valueOf()
+    const restante = moment.duration(diff_millis).humanize()
```
algo super menor, pero creo que sería mejor `remaining` ya que se estan usando variables en inglés en el resto del código.

sebasjm replied:
:+1:

gianu commented on src/js/modules/offline.js:
```diff
@@ -0,0 +1,30 @@
+       .register('service-worker.js')
+       .then(function() {
+          // eslint-disable-next-line no-console
+          //  console.log('Service Worker Registered')
```
Si esto va a estar comentado, se podría quitar todo el `then`.

sebasjm committed:
fix typo

sebasjm committed:
agrego dependencias a la tarea de pwa-build

esto estaba provocando que se genere el archivo service-worker incorrecto: con referencia a archivos js.map
como consecuencia no se actualizaba no se actualizaba el service-worker instalado en el cliente cuando se subía una nueva versión

sebasjm committed:
fix lint

durancristhian committed:
Merge branch 'dev' into add-notification
